.circleci/config.yml

```diff
@@ -12,22 +12,26 @@ jobs:
 
       - restore_cache:
           keys:
-            - v1-dependencies-{{ checksum "yarn.lock" }}
-
-      - run: yarn install --frozen-lockfile
+            - v4-dependencies-{{ checksum "yarn.lock" }}
+            - v4-dependencies-
+      - run: corepack use yarn@4.2.2
+      - run: yarn set version berry
+      - run: yarn install --immutable
 
       - save_cache:
           paths:
             - node_modules
-          key: v1-dependencies-{{ checksum "yarn.lock" }}
+          key: v4-dependencies-{{ checksum "yarn.lock" }}
 
       - run: yarn lint
 
+      # test if dist is correctly generated from src
+      - run: yarn build && git status --porcelain
+
       # unit test
       - run: yarn test
 
       # publish code coverage results
       - run: bash <(curl -Ls https://coverage.codacy.com/get.sh)
 
-      # test if dist is correctly generated from src
-      - run: yarn build && git status --porcelain
```

Comment:
Why do we not need this anymore?

Reply:
This was not removed, but rather, it was placed above the yarn test command for more consistent test runs with less flake.